model/otel/sampling.yaml

```diff
@@ -0,0 +1,49 @@
+groups:
+  - id: sampling.common
+    type: attribute_group
+    brief: 'Semantic conventions describing sampling in OpenTelemetry.'
+    prefix: sampling
```

Comment:
Since it's otel-related attribute, we should consider putting it in the `otel.` namespace

Reply:
I've added text to argue in favor of "sampling" because this "sampling.priority" is already defined.